src/hooks/useRoomIdsStorage.ts

```diff
@@ -0,0 +1,30 @@
+import { useEffect } from 'react'
+import { useStore } from '../components/Store'
+import { StorageKey } from '../utils/types'
+
+export default function useRoomIdsStorage() {
```

Comment:
This hook makes sure the list of room ids is always up to date. It ignores resetting (a.k.a. logging out).